### DISCUSSION TITLE
付费影视不能给到最高画质

### DISCUSSION THREAD

**Discussion by YFTX2015** [ywmoyue/biliuwp-lite#850] [bug]
### Bug 描述

拿盒中之海这部动画电影举例，原始网站最高给到4K，这边最高只有720P。

### 操作步骤

```plain text
在app内搜索盒中之海，以页面显示的情况做出此判断。
```

### 期望结果

能够跟隔壁哔哔(22与33的秘密基地）一样，使第三方客户端也能给到付费视频最高画质（用户已付费的情况下），不论是播放还是下载。

### 截图

![屏幕截图(6)](https://github.com/user-attachments/assets/8122f2a9-bf3e-4c48-aca6-49eb0df4a0e6)

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
确实有问题，我定位下
